var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -29,3 +30,6 @@ class PyBiomFormat(PythonPackage):
     depends_on("py-scipy@1.3.1:", type=("build", "run"))
```

Comment:
Future isn't needed in this new version

Reply:
WDYM?

Reply:
Right now `py-future` is listed as a dependency for all versions, but 2.1.15 doesn't depend on py-future